tests/unit/test_models.py

```diff
@@ -104,108 +105,138 @@ def test_validate_grammar_recursive():
             """
             control: a string
             grammar_str:
-              on amd64: another string
-              else:
-                to arm64: this other thing
+              - on amd64: another string
+              - else:
+                - to arm64: this other thing
+                - else fail
             grammar_strlist:
-              to amd64,arm64:
-                on riscv64:
+              - to amd64,arm64:
```

Comment:
I don't think we want to support this anymore, we want the full specification:

    - on <arch>,... to <target-arch>,...:

Reply:
Plain "to: <selector>" could be useful to specify stage packages for the runtime environment (unless we want to be restrictive about the actual build system arch), as in `build on anything-I-dont-care to arm64: arm64-pkg`.

Reply:
During our conversations with Gustavo we found that a simple to is not descriptive enough, we need to have at least a parent on. So this could work:

    - on amd64:
        - to arm64:
             # replace with real package name
            - gcc-arm64
            - libncurser:arm64
        - to armhf:
             # replace with real package name
            - gcc-armhf
            - libncurser:armhf
        - to armhf,arm64:
             # just an example of a multi-match
             - libncurses-dev